example/discovery.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:dbus/dbus.dart';
+import 'package:bluez/bluez.dart';
+
+import 'package:convert/convert.dart';
+
+void process_ble(var device) {
+  var mac = device.address;
+  int rssi = device.rssi;
+  print('\nName: ${device.name}');
+  print('Address: ${mac}\nRSSI: ${rssi}');
+
+  if (device.manufacturerData.isNotEmpty) {
+    var mfg;
+    int mfgId;
+    mfg = hex.encode((device.manufacturerData.values.toList())[0]);
+    mfgId = (device.manufacturerData.keys.toList())[0];
```

Comment:
Please change to `var mfg = ...` / `var mfgId = ...` instead of the variable definitions above.